tests/e2e/logging/test_openshift-logging.py

```diff
@@ -67,6 +73,14 @@ def teardown(cbp_obj, sc_obj):
     """
     The teardown will uninstall the openshift-logging from the cluster
     """
+    cur_dir_files = run_cmd("ls")
+    if 'OperatorSource_Images_Labels.txt' in cur_dir_files:
```

Comment:
We can remove the need for `ls` if we use `os.path.isfile()`
```suggestion
    if os.path.isfile('OperatorSource_Images_Labels.txt')
```

Reply:
Done